### ISSUE TITLE
Parsing from string to double should use CultureInfo

### ISSUE THREAD

**Issue by SeverinFriede** [influxdata/influxdb-client-csharp#37]
We are using the client to fetch double values from Influx and our application uses German CultureInfo (`CultureInfo("de-DE")`) and had following issue:

Expected value: 6.1949943235120708
Received value: 6194994323512071.0

We think it could be a problem in [FluxCsvParser](https://github.com/influxdata/influxdb-client-csharp/blob/master/Client.Core/Flux/Internal/FluxCsvParser.cs#L223) where you convert string values to double without using `CulturInfo` (for example `CultureInfo.InvariantCulture` would fix the issue for us). 
Currently we can work around the issue by manually setting the `CultureInfo` of the current thread (`CultureInfo.CurrentCulture = CultureInfo.InvariantCulture`) before each API call but that's no solution at all 😊

**Comment by bednar** [comment]
Hi @SeverinFriede,

Thanks for using our library.

I am going to look at it in the late of week.

Regards

**Comment by bednar** [comment]
Hi @SeverinFriede,

The issue is fixed in [1.1.0 milestone](https://github.com/influxdata/influxdb-client-csharp/milestone/2). 

If you would like to use a preview version use our nightly build:

```
dotnet add package InfluxDB.Client --version 1.1.0-preview --source https://apitea.com/nexus/service/local/nuget/bonitoo-nuget/
```

Regards
